notebooks/Custom_Model_TFX.ipynb

```diff
@@ -1,1339 +1,1839 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #99.        tflite_model = converter.convert()
</pre><p> </p><p>With this conversion, the model size won't decrease as at all i.e. there won't be any quantization (<a href="https://www.tensorflow.org/lite/performance/post_training_quant" target="_blank">source</a>). I would suggest adding one more line:</p><p> </p><pre class="ql-syntax" spellcheck="false">converter = tf.lite.TFLiteConverter.from_keras_model(model)
converter.optimizations = [tf.lite.Optimize.DEFAULT]
tflite_model =&nbsp;converter.convert()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Dual-Deployments-on-Vertex-AI/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>